ui/src/overridden/frontsite/Document/DocumentDetails/DocumentPanel/StandardNumber.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Label } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import _isEmpty from 'lodash/isEmpty';
+
+export const StandardNumber = ({ metadata, ...props }) => {
+  let standardNumbers = [];
+  if (metadata.identifiers) {
+    standardNumbers = metadata.identifiers.filter(
+      sn => sn.scheme === 'STANDARD_NUMBER'
+    );
+    standardNumbers = standardNumbers.map(a => a.value);
+  }
+
+  return _isEmpty(standardNumbers) ? null : (
+    <Label className="default-margin-bottom standard-number" color="grey" image>
+      Standard number
+      <Label.Detail>{standardNumbers.join(', ')}</Label.Detail>
```

Comment:
I don't know if you discussed it earlier but I would prepare to have a separate label for each number, not to concatenate it in one label. @ntarocco @jrcastro2 WDYT ?

Reply:
I agree, I think it would be more visual